buildscripts/incremental/test.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+source activate travisci
+# Ensure that the documentation builds without warnings
+cd $TRAVIS_BUILD_DIR/docs
+if [ "$BUILD_DOC" == "yes" ]; then make SPHINXOPTS=-W clean html; fi
```

Comment:
This is directly moved from .travis.yml. No attempts have been made to change / improve the logic. If something is wrong with the above, let's fix that separately.